cernopendata/config.py

```diff
@@ -190,3 +190,16 @@
     'APP_SQLALCHEMY_DATABASE_URI',
     'postgresql+psycopg2://localhost/cernopendata'
 )
+
+# DataCite
+# ========
+#: DataCite API - URL endpoint.
+PIDSTORE_DATACITE_URL = "https://mds.datacite.org"
+#: DataCite API - Disable test mode (we however use the test prefix).
+PIDSTORE_DATACITE_TESTMODE = False
+#: DataCite API - Prefix for minting DOIs in (10.5072 is a test prefix).
+PIDSTORE_DATACITE_DOI_PREFIX = "10.5072"
+#: DataCite MDS username.
+PIDSTORE_DATACITE_USERNAME = "CERN.OPENDATA"
+#: DataCite MDS password.
+PIDSTORE_DATACITE_PASSWORD = "CHANGE_ME"
```

Comment:
I hope the password will be read from an environment variable or something? Would be good to add an example test value to `docker-compose.yml` perhaps?